packages/mobile/src/account/saga.ts

```diff
@@ -69,7 +69,8 @@ export function* getPincode(withVerification = true) {
     const pin = yield new Promise((resolve, reject) => {
       navigate(Screens.PincodeEnter, {
         onSuccess: resolve,
-        onFail: reject,
+        // Looks like this param is not used/needed anymore
+        // onFail: reject,
```

Comment:
@i1skn TypeScript caught that this param wasn't used anymore. Could you confirm it's the intended behaviour?

Reply:
It seems it is unused! Nice to see new reactnavigation v6 in action 💪 Also, `reject on line 69 probably would be unused as well.